docs/framework/data/adonet/ef/aggregate-functions-sqlclient-for-entity-framework.md

```diff
@@ -6,39 +6,222 @@ ms.assetid: 03303f01-b591-4efc-9875-f9c608edff0b
 # Aggregate Functions (SqlClient for Entity Framework)
 The .NET Framework Data Provider for SQL Server (SqlClient) provides aggregate functions. Aggregate functions perform calculations on a set of input values and return a value. These functions are in the SqlServer namespace, which is available when you use SqlClient. A provider's namespace property allows the Entity Framework to discover which prefix is used by this provider for specific constructs, such as types and functions.  
   
- The following table shows the SqlClient aggregate functions.  
+ The following are the SqlClient aggregate functions.  
+
+## `AVG(expression)`
```

Comment:
wondering if it would be better not to inline code the section names. The font looks a bit small on review with this formatting.